src/stores/Binding/Store.ts

```diff
@@ -1043,13 +1049,27 @@ const getInitialState = (
                     ? state.resourceConfigs[targetBindingUUID]
                     : null;
 
+                const targettedResourceConfig =
```

Comment:
This variable could simply be `targetResourceConfig` since it is the compliment of `targetBindingUUID`.